src/Integration.Vsix/SonarLintTagger/TextBufferIssueTracker.cs

```diff
@@ -42,7 +43,7 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     /// See the README.md in this folder for more information
     /// </para>
     /// </remarks>
-    internal class TextBufferIssueTracker : IIssueTracker
+    internal class TextBufferIssueTracker : IIssueTracker, ITagger<IErrorTag>, IDisposable
```

Comment:
How the taggers are actually disposed? (since they don't implement IDisposable -- so what actually calls the Dispose?)

Reply:
Not sure. The editor might check if they are disposable. Otherwise, it'll be when they are garbage collected.

Reply:
Yep, TaggerAggregator checks if they're disposable

![image](https://user-images.githubusercontent.com/60586879/92392617-d5699200-f11e-11ea-8a07-197086aed29a.png)